### PR TITLE
Osram/Sylvania: Add vendor specific functions

### PR DESCRIPTION
**Pull request by qosmio**
NOTE: Will require updated zcl-id and zcl-packet defs:

https://github.com/qosmio/zcl-id/commit/859d6575cac060c79c73405ea0e5c3760616e8a6

https://github.com/qosmio/zcl-packet/commit/f74d92ce6f8da88723e3c9174d347e77c96f2c75

Two new functions:

* osram_set_transition: Allows setting a default transition time
whenever lights are turned back on manually.

* osram_remember_state: Allows the end device to remember its last
brightness/hue/color/etc state when light is turned on manually.
Currently it resets to default.

Also adding options to the incoming message ignore list.
  - fz.ignore_genIdentify_change
  - fz.ignore_diagnostic_change
  - fz.ignore_genscenes_change

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -1519,14 +1519,54 @@ const devices = [
...
+        ]),
+        configure: (ieeeAddr, shepherd, coordinator, callback) => {
+            const device = shepherd.find(ieeeAddr, 3);
+            const cfgOnOff = {direction: 0, attrId: 0, dataType: 16, minRepIntval: 3, maxRepIntval: 1000, repChange: 1};
+            const cfgLevel = {direction: 0, attrId: 0, dataType: 32, minRepIntval: 3, maxRepIntval: 1000, repChange: 1};
+            const actions = [
+                (cb) => device.bind('genOnOff', coordinator, cb),
+                (cb) => device.foundation('genOnOff', 'configReport', [cfgOnOff], foundationCfg, cb),
+                (cb) => device.bind('genLevelCtrl', coordinator, cb),
+                (cb) => device.foundation('genLevelCtrl', 'configReport', [cfgLevel], foundationCfg, cb),
```
Comment (Koenkk) [inline devices.js]: Same as https://github.com/Koenkk/zigbee-shepherd-converters/pull/351#issuecomment-473575855

Comment (qosmio) [reply devices.js]: Just reverted those changes. Only left in ignore options and the Osram functions.

**Comment by Koenkk** [comment]
Thanks! Could you also make a PR to update the docs here: https://github.com/Koenkk/zigbee2mqtt.io/blob/dev/information/mqtt_topics_and_message_structure.md#device-specific ?

**Comment by qosmio** [comment]
Will do! Was just looking at this last night and remembered I should create one for Osram. Thanks!

**Pull request merged**